system-int-test/src/cmr/system_int_test/utils/index_util.clj

```diff
@@ -68,6 +69,18 @@
                     :throw-exceptions false})]
     (is (= 200 (:status response)) (:body response))))
 
+(defn doc-present?
+  "If doc is present return true, otherwise return false"
+  [index-name type-name doc-id]
+  (let [response (client/get
+                  (format (str (url/elastic-root) "/%s/%s/_search?_id=%s") index-name type-name doc-id)
+                  {:throw-exceptions false
+                   :headers {transmit-config/token-header (transmit-config/echo-system-token)}
```

Comment:
Elasticsearch doesn't use ECHO token headers. You may want to replace these calls with a elastisch call.

Reply:
Once the search is implemented this will no longer be needed.